documentation/docs/changelog.md

```diff
@@ -5,6 +5,61 @@ slug: changelog.html
 ---
 
 
+### [Unreleased]
+
+Note: This changelog mentions the changes which are not yet released in stable version.
+      You can still use snapshot release to tryout changes and provide us your's valuable
+      feedback. Docs updates are available at kotest.io though.
+
+#### Features / Improvement
+* Added new matcher for DayOfWeek in `kotest-assertion-clock` module. #2124
+* Added factory method to simplify creating new matchers. #2122
+* Added method in `Exhaustive` to create a new `Exhaustive` which will be a cartesian product of given two `Exhaustive`. #2120
+* Added support for writing test inside a object instead of creating a class. #2097
+* Added suspend version of `shouldCompleteWithin`, `shouldCompleteBetween` and `shouldTimeOut`. #2107
+* Added `kotest-extensions-wiremock` module for managing lifecycle of `WireMockServer` in Kotest test. #2108
+* Upgrade `klock` dependency to 2.0.6 and added `browser`, `nodejs`, `linuxX64`, `mingwX64`, `macosX64`, `tvos`,
+  `iosX64`, `iosArm64` and `iosArm32` platform targets for `kotest-assertions-klock`. #2116
+* Run eventually one final time if iterations is one and delay is greater than the interval #2105
+* Updates `withEnvironment` function to ignore environment variable key case sensitivity on Windows platform. #2099
+* Some improvement around `eventually`
+  1. Makes EventuallyPredicate<T> a type alias instead of interface for better user experiance.
+  2. Update failure message to inform the user about failure of given EventaullyPredicate.
+  3. Adds an overload of `eventually` which does not takes `EventuallyPredicate` so that it give feel
+     of `util` function. #2046
+
+
+
+#### BugFixes.
```

Comment:
```suggestion
#### Bugfixes
```

To match the format of the previous entries

Reply:
Corrected